lib/redcloth3.rb

```diff
@@ -593,7 +593,7 @@ def block_textile_quotes( text )
           bq,content = $1, $2
           l = bq.count('>')
           if l != indent
-            quotes << ("\n\n" + (l>indent ? '<blockquote>' * (l-indent) : '</blockquote>' * (indent-l)) + "\n\n")
+            quotes << ("\n\n" + (l>indent ? '<blockquote class="icon icon-quote2">' * (l-indent) : '</blockquote>' * (indent-l)) + "\n\n")
```

Comment:
Line is too long. [138/100]<br>Surrounding space missing for operator '>'.<br>Surrounding space missing for operator '-'.